stable/yugabyte/templates/service.yaml

```diff
@@ -426,11 +426,37 @@ spec:
         {{- if $root.Values.tls.enabled }}
         - name: {{ $root.Values.oldNamingStyle | ternary (printf "%s-yugabyte-tls-cert" .label) (printf "%s-%s-tls-cert" (include "yugabyte.fullname" $root) .label) }}
           secret:
+            {{- if not $root.Values.tls.certManager.enabled }}
             secretName: {{ $root.Values.oldNamingStyle | ternary (printf "%s-yugabyte-tls-cert" .label) (printf "%s-%s-tls-cert" (include "yugabyte.fullname" $root) .label) }}
+            {{- else }}
+            secretName:  {{ $root.Values.oldNamingStyle | ternary (printf "yugabyte-%s-tls" $service.label) (printf "%s-%s-tls" (include "yugabyte.fullname" $root) $service.label) }}
```

Comment:
What do you think about re-using the same `secretName` regardless of whether or not `certManager.enabled` is true?